### ISSUE TITLE
Use TriliumNext Branding

### ISSUE THREAD

**Issue by eliandoran** [TriliumNext/Notes#195]
### Describe feature

Whenever "Trilium Notes" is used, replace it with either:

* TriliumNext Notes
* Trilium Notes Next

To be discussed internally on the [general channel](https://matrix.to/#/#triliumnextgeneral:matrix.org).

**Comment by eliandoran** [comment]
@TriliumNext/everyone , let's decide what's the best "product name" for the first version.

**Comment by maphew** [comment]
**Trilium** - short is nice 🙂 

**Trilidium** - proposed by @meichthys somewhere I've lost track of. Initially I was cool on this name, but it's grown on me and I quite like it now.

**Metril** - or 'My Trilium' is what I call my personal collection of scripts for migrating from others sources to Trilium Notes. I'd happily give it up for the cause.

**Comment by chesspro13** [comment]
> **Trilium** - short is nice 🙂
> 
> **Trilidium** - proposed by @meichthys somewhere I've lost track of. Initially I was cool on this name, but it's grown on me and I quite like it now.
> 
> **Metril** - or 'My Trilium' is what I call my personal collection of scripts for migrating from others sources to Trilium Notes. I'd happily give it up for the cause.

I second Trilidium. Nice to say, nice to type.

Have we heard what zadam's thoughts on this subject are?

**Comment by eliandoran** [comment]
> @eliandoran What do you mean by product name? Are you referring to the name of the release? I know some projects provide a different name for every release.

@meichthys , the product name would be the name of the application or the software that we are developing. We already decided that the org is TriliumNext but now we would need to rename it from "Trilium Notes" to something. Personally I believe it should be TriliumNext-related, but we simply can't call it "TriliumNext" since it would no longer indicate that it's a note-taking application.

The product name is used for example:

In the contextual menu:
<img width="482" alt="image" src="https://github.com/user-attachments/assets/950a58d0-5e06-415c-bfd5-250aa887c75d">

In the about modal:
<img width="788" alt="image" src="https://github.com/user-attachments/assets/4fae1a1c-97f1-4e4e-9acc-ac371c62a5f2">

And during on-boarding/first start and a few more uses.

**Comment by CrO2Cl2** [comment]
Trilidium sounds absolutely awesome

**Comment by Alumniminium** [comment]
Trilidium sounds absolutely retarded in my opinion. 

Why do we need to rebrand Trilium? Keep it. Zadam chose that name and we should carry on the torch.

what the f...?!

**Comment by perfectra1n** [comment]
Because Zadam isn’t okay with giving up the name “Trilium” yet.

**Comment by eliandoran** [comment]
> Trilidium sounds absolutely retarded in my opinion.
> 
> Why do we need to rebrand Trilium? Keep it. Zadam chose that name and we should carry on the torch. TriliumNext is fine to indicate the development moved to new maintainers but rebranding the entire thing is ridiculous.
> 
> We are not a business whitelabeling a product, we are taking over maintenance of an existing product with a name.
> 
> what the f...?!

@Alumniminium , although it might not seem like a good idea, renaming is something we need to do. The upstream repo (zadam's) is still functioning and is under maintenance of zadam. We are forking a repo, so keeping two projects with the exact same name and but with different features and states of development can be quite confusing to the users.

And @perfectra1n is right, Zadam has not "given us the torch", unfortunately. It was forked on our accord and we are not officialy in charge of taking over zadam's work (at least not yet).

This is also important for a very simple reason: if you happen to have both the fork and the upstream installed, how do you differentiate between the two? Sure, you can have different version numbers but you would still need to know the versioning scheme and hope that these versions never collide between the two projects.

Also I would kindly ask you to be more reasonable with your words.

**Comment by maphew** [comment]
Zadam's words on the topic from the thread referenced above, emphasis mine:

> I'd recommend simply forking this repository and going from there, the repository contains everything needed. **I don't have any copyright or trademark on the Trilium name, the icon etc, although I think it would be practical to differentiate the new fork somehow.** If some fork gains traction, I'm happy to feature it prominently on the README, next to the notice on the maintenance status.
> ...
> There were some mentions in this thread about respecting my wishes, I appreciate that, but I want to stress that this concerns the zadam/trilium repository only. I don't claim any decision authority in the new organization

**Comment by root-hal9000** [comment]
I am just catching up with everything on the project today (including posting those minutes, which I dropped the ball on), but I just wanted to express my support for **NeoTrilium** as the name - someone suggested it on the chat (I forget who). I think it says what it is (new trilium), and rolls off the tongue a little easier than trilidium (although I also like that one).

As a side note, we have preemptively purchased these domains to hold on to just in case:
triliumnext.com
neotrilium.org

**Comment by rauenzi** [comment]
Of the suggestions so far I think `NeoTrilium` would be the one I like the least. I think `Trilidium` makes the most sense so far logically speaking, but I'm not _personally_ married to any of the suggestions. If we want to ensure we differentiate while still holding the namesake, we could also go the same route paperless did and become `trilium-ng` where `ng` stands for `next generation`. I would rank that one slightly above plain `Trilium Next` but below `Trilidium`

**Comment by chesspro13** [comment]
We've also got trilidium.com

**Comment by root-hal9000** [comment]
I am actually warming up to trilidium more myself. I just realized I liked it less when discussing it in the chat because the font rendering made the lower case I-L-I combination hard to read. made me feel a little dyslexic :rofl:

**Comment by hansklepitko** [comment]
Hello from the silent side. I've been using Trilium for a long time, and I believe Trilium NG and NeoTrilium make the most sense, as there may be many users that use Trilium similarly to me but are unaware of a big fork. Changing the name too much may not be the ideal way in terms of migration for old users and catching up with the changes.

**Comment by rom1dep** [comment]
I just posted a message on Matrix which I think is on point for this discussion (and why I'm pasting it here). tl;dr, I don't think this bikeshedding about the name matters for the moment: It's ok not to commit on a new name when the project hasn't taken off yet (but that's just my opinion anyhow).

> I shared my perspective previously, and I will do it again here since I couldn't attend the meeting for family reasons. I think we should all realize that there are hundreds, probably thousands (or more!) daily users of Trilium Notes who are silently happy about the software in its current shape and form (including its author).
> 
> This is day one of "post Trilium", and people are understandably very excited about what may come next, want to leave a mark, want to propose clever names, want to pull their favourite piece of software architecture and framework, etc. "Trilium Next", as a name, is more a placeholder in search for a meaning than the expression of clear goals and new vision, and I like it this way, this is where we are right now: trying to figure out the next steps. Essentially, variations around this theme (like trilium ng/neo trilium/trilidium/…) do not matter at the moment. What matters is that the many silent users can see and understand that there is a nascent org who set for itself to continue where Trilium stopped.
> 
> And as much as I am supportive of what's going on here, I (and the thousand others who depend on Trilium daily) won't take this new project favourably nor to the same level of seriousness and trust as the original one. Not until it has continuously delivered something that they like and that works to the same high standards as set by @zadam .
> 
> That's why I've been pitching this idea of a two-paced "Trilium Next": one maintenance branch maximizing stability and compatibility with Trilium, as a "gateway drug" towards the second branch, packing novelties and more adventurous and disruptive changes (and again, most people are not asking for that!).
> 
> Anyhow, that's the condition I see for people to build trust in this new leadership and to get on board with the new app.
> If this org just wants to build a new note taking app with no consideration for the existing userbase, I'm sorry to be the one pointing out that there are many, many, well founded apps out there, and that the competition is fierce. Those silent users will contemplate a tough choice very soon: "should I abandon trilium and migrate to the new hot thing? Or will this new org be a safe refuge?". I really hope for the second.

**Comment by eliandoran** [comment]
I think we exceeded a bit the scope of this issue. I created it simply to be able to distinguish the soon-to-be-released TriliumNext application from "Trilium Notes", in order not to cause confusion. Apologies if I was not explicit enough in how the issue was written.

Although suggestions for a new name are much appreciated, it is not the time yet and as discussed during the first steering meeting.

Due to lack of consensus, we'll stick to using "TriliumNext Notes" instead of "Trilium Next" throughout the application.

I made sure to keep a reference to this issue thread in https://github.com/orgs/TriliumNext/discussions/3 and maybe we'll revisit it once the project becomes stable.
